pkg/profile/artifact_test.go

```diff
@@ -147,7 +162,8 @@ var _ = Describe("Profile", func() {
 		When("the artifact exists", func() {
 			var (
 				gitRes                       *sourcev1.GitRepository
-				helmRes1, helmRes2           *helmv2.HelmRelease
+				helmRep                      *sourcev1.HelmRepository
+				helmRes1, helmRes2, helmRes3 *helmv2.HelmRelease
```

Comment:
minor nit: I think we can mdoify `helmRes2` instead of adding a new `helmRes3`. Two exist just for testing that it supported multiple `HelmChart` kinds, so the 2nd can be a remote chart :smile:

Reply:
Yeah, sure, no problem. The BBD test cases are a bit confusing to be honest. :D It's not readily apparent which variables are safe to be re-used.

Reply:
happy to pair to go over if if you want :smile: